packages/sources/jpegd/src/endpoint/punks.ts

```diff
@@ -11,10 +11,11 @@ export interface ResponseSchema {
 
 export const inputParameters: InputParameters = {
   block: {
-    required: true,
+    required: false,
     description: 'The block number for which information is being queried',
     aliases: ['blockNumber', 'blockNum'],
-    type: 'number',
+    type: 'string',
```

Comment:
If the input can be multiple types, you can just remove this line. We don't have a way to define multiple types, but it will not pass validation if the type doesn't match the value passed in.